ios/RNCarPlay.m

```diff
@@ -93,6 +101,41 @@ - (dispatch_queue_t)methodQueue
     return dispatch_get_main_queue();
 }
 
+
+-(UIImage *)imageWithTint:(UIImage *)image andTintColor:(UIColor *)tintColor {
+    UIImage *imageNew = [image imageWithRenderingMode:UIImageRenderingModeAlwaysTemplate];
+    UIImageView *imageView = [[UIImageView alloc] initWithImage:imageNew];
+    imageView.tintColor = tintColor;
+    UIGraphicsBeginImageContextWithOptions(imageView.bounds.size, NO, 0.0);
+    [imageView.layer renderInContext:UIGraphicsGetCurrentContext()];
+    UIImage *tintedImage = UIGraphicsGetImageFromCurrentImageContext();
+    UIGraphicsEndImageContext();
+    return tintedImage;
+}
+
+- (UIImage *)imageWithSize:(UIImage *)image convertToSize:(CGSize)size {
+    UIGraphicsBeginImageContext(size);
+    [image drawInRect:CGRectMake(0, 0, size.width, size.height)];
+    UIImage *destImage = UIGraphicsGetImageFromCurrentImageContext();
+    UIGraphicsEndImageContext();
+    return destImage;
+}
+
+- (UIColor *)colorFromHexString:(NSString *)hexString {
```

Comment:
Could we not use RCTConvert color:UIColor and cast that to rgb?

Reply:
My strength does not lie in objective-c land I'm afraid so if you feel there is a better way to do this, please let me know. Happy to try the above out.

Reply:
@birkir switched over the use `RCTConvert` as you said and made the use of `processColor` optional on the front end, so the user can specify `yellow` / `#ff0000` or `processColor('pink')`. Keeps the API flexible on the TSX side and follows the pattern of resolving image sources for the user etc.